src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_up_commands.py

```diff
@@ -130,8 +130,7 @@ def test_webapp_up_python_e2e(self, resource_group):
         ])
 
         self.cmd('webapp config show', checks=[
-            JMESPathCheck('linuxFxVersion', result['runtime_version']),
-            JMESPathCheck('tags.cli', 'webapp_up'),
```

Comment:
can we just update this validation to tags.cli [] instead of removing the lime completely?

Reply:
this update needs to be made at 2 others places on this same file. Please make sure you run all tests in this file with --live flag